13/umbraco-deploy/release-notes.md

```diff
@@ -20,33 +20,71 @@ Under each major version, you can find details about minor and patch releases fo
 
 <summary>Version 13</summary>
 
-[**13.0.0-rc3**](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F13.0.0)
-(November 28th 2023)
+[**13.0.1**](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F13.0.1) **(December 21th 2023)**
+
+* Fixes the display of the selected schedule date on queue for transfer.
+* Fixes parsing property values within Nested Content and Block List that were previously saved by the Contrib value connectors.
+* Fixed incorrectly including media files in export when 'Content files' wasn't selected.
+* Add maximum file size validation to import file upload.
+
+[**13.0.0**](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F13.0.0) **(December 14th 2023)**
+
+* All items from the 13.0.0 RCs and latest 12.2 features.
+
+[**13.0.0-rc5**](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F13.0.0) **(December 12th 2023)**
+
+* Align with latest CMS RC.
+
+[**13.0.0-rc4**](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F13.0.0) **(December 11th 2023)**
+
+* Add weight -100 to Deploy package migration (ensuring Deploy database tables are available before other package migrations are executed).
+* Add fluent API for Deploy webhooks.
+
+[**13.0.0-rc3**](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F13.0.0) **(November 29th 2023)**
 
 * Added optional deployment of webhooks as part of schema updates.
 * Added Deploy specific webhook events.
 
-[**13.0.0-rc1**](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F13.0.0)
-(November 6th 2023)
+[**13.0.0-rc2**](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F13.0.0) **(November 15th 2023)**
 
-* Compatibility with Umbraco 13
+* Exclude automatic relation type aliases (used for reference tracking) from deployment.
+* Fix cyclic dependency between configuring `DeploySettings` and `ConnectionStrings`.
+
+[**13.0.0-rc1**](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F13.0.0) **(November 6th 2023)**
+
+* Compatibility with Umbraco 13:
   * See full details of breaking changes under the [version specific upgrade guide](upgrades/version-specific.md).
+  * Update Richtext value connector to handle references in blocks.
 
 </details>
 
 <details>
 
 <summary>Version 12</summary>
 
-[**12.1.0**](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F12.1.0) **(December 11th  2023)**
+[**12.2.1**](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F12.2.1) **(December 21th 2023)**
```

Comment:
Should the release notes be duplicated for all previous versions to their respective documentation versions or should we keep the release notes on the latest version (and link to this document instead)?

Reply:
a history should be kept. So when checking for v12 the release notes there should be all versions to see what changes or fixes have been done between minors and patches.

Reply:
oh, wait a bit, do you mean here for v13 there should be only release notes for v13 or for v12 and below as well? 

If you are asking about this, this is something that the DXP needs to decide, so please ask @AndyButland and the other as well.